app-tests/cra-layout-admin/src/App.js

```diff
@@ -1,16 +1,16 @@
 import React from 'react';
-import { Admin } from '@espressive/cascara';
+import { AdminStructure } from '@espressive/cascara';
 
 const App = () => {
   return (
-    <Admin
+    <AdminStructure
       header={
-        <Admin.Header>
+        <AdminStructure.Header>
           <h1>{process.env.REACT_APP_NAME}</h1>
-        </Admin.Header>
+        </AdminStructure.Header>
```

Comment:
This API change should match the API change being made in the next-layout-admin template for the header.